projects/packages/search/src/dashboard/components/dashboard/connection-page.scss

```diff
@@ -1,6 +1,16 @@
 @import '@automattic/jetpack-base-styles/style';
 
 .jp-search-dashboard-connection-screen {
+	background: var(--jp-white-off);
+	align-items: center;
+	display: flex;
+	width: 100%;
+
+	// Targets AdminSectionHero component
```

Comment:
Is there a class name we could target instead?

Reply:
That's not possible, I'm afraid. [AdminSectionHero](https://github.com/Automattic/jetpack/blob/master/projects/js-packages/components/components/admin-section/hero/index.jsx#L19) uses obfuscated class names:

<img width="369" alt="Screen Shot 2022-04-21 at 4 29 12 PM" src="https://user-images.githubusercontent.com/4044428/164567247-25456dc3-5c9d-4f5b-b4c4-46c7c7f70573.png">

I'll add some code comments pointing this out and warning contributors on this brittle style rule.